packages/mco/hooks/mco-resources.ts

```diff
@@ -83,6 +85,26 @@ export const getPlacementDecisionsResourceObj = (
   optional: true,
 });
 
+export const getSubscriptionResourceObj = (props?: NamespacedObjectType) => ({
+  cluster: HUB_CLUSTER_NAME,
+  ...(!!props?.name ? { name: props?.name } : {}),
+  ...(!!props?.namespace ? { namespace: props?.namespace } : {}),
```

Comment:
same at other places in this file...
```suggestion
export const getSubscriptionResourceObj = (props?: NamespacedObjectType) => ({
  cluster: HUB_CLUSTER_NAME,
  ...(!!props?.name ? { name: props.name } : {}),
  ...(!!props?.namespace ? { namespace: props.namespace } : {}),
```
OR,
```suggestion
export const getSubscriptionResourceObj = ({name, namespace}) => ({
  cluster: HUB_CLUSTER_NAME,
  ...(!!name ? { name } : {}),
  ...(!!namespace ? { namespace } : {}),
```